starport/services/networkbuilder/networkbuilder.go

```diff
@@ -169,6 +170,11 @@ func (b *Builder) Init(ctx context.Context, chainID string, source SourceOption,
 
 	// set options.
 	o := &initOptions{}
+
+	// set the default home
+	o.homePath = filepath.Join(homePath, chainID)
```

Comment:
We can have a `newInitOptions` that returns `initOptions` with this default value and reuse it.